Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -259,11 +264,11 @@ private class TopSitesLayout: UICollectionViewLayout {
 
     private func setupForOrientation(orientation: UIInterfaceOrientation) {
         if orientation.isLandscape {
```

Comment:
I... don't think it is. Basically we want to show 4 in portrait on tablets and 5 in landscape, but neither the horizontal (or vertical) size class changes there. I did switch the order of the checks here so that regardless of orientation, if a tablet moves into horizontal-size-class == .Compact, we'll switch to showing 3.